volmaster/volmaster/cli.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/codegangsta/cli"
 )
 
+// version is provided by build
+var version = ""
```

Comment:
Think we need to do a dockerversion/netctl version like thing here? I can do it in another patch, but it'll kill the `go get` notion.